.circleci/config.yml

```diff
@@ -157,6 +165,19 @@ jobs:
           name: Invalidate CloudFront caches
           command: aws cloudfront create-invalidation --distribution-id EPTW7F3CB566V --paths "/*"
   
+  make-release:
+    macos:
+      xcode: "12.2.0"
+    working_directory: ~/purchases-ios/
```

Comment:
I believe this is the default

Reply:
I thought so, but I had run into issues with other jobs when removing this